torch/testing/_internal/distributed/rpc/combining_ddp_with_rpc_test.py

```diff
@@ -0,0 +1,191 @@
+import random
+import sys
+
+import torch
+import torch.distributed as dist
+import torch.distributed.autograd as dist_autograd
+from torch.distributed.optim import DistributedOptimizer
+import torch.distributed.rpc as rpc
+from torch.distributed.rpc import RRef
+from torch.distributed.rpc import TensorPipeRpcBackendOptions
+from torch.nn.parallel import DistributedDataParallel as DDP
+import torch.optim as optim
+from torch.testing._internal.distributed.rpc.rpc_agent_test_fixture import (
+    RpcAgentTestFixture,
+)
+from torch.testing._internal.common_distributed import (
+    TEST_SKIPS,
+)
+from torch.testing._internal.dist_utils import (
+    dist_init,
+)
+
+NUM_EMBEDDINGS = 100
+EMBEDDING_DIM = 16
+
+
+class HybridModel(torch.nn.Module):
```

Comment:
Hey @gcramer23, there might be some misunderstanding, the two onboarding tasks mentioned in the onboarding doc are the following. Did you get this task from someone else?

1. Add distributed example for CUDA RPC (https://pytorch.org/tutorials/recipes/cuda_rpc.html) to existing PyTorch CI tests.

* https://github.com/pytorch/pytorch/issues/46321 (CUDA RPC)

2. Add a tests to test_c10d.py to explicitly cover DDP on mixed dense-sparse models. Currently we have a sparse-only module tests.

* https://github.com/pytorch/pytorch/issues/54445


The second task should only use DDP (gloo backend) for mixed dense and sparse tensors.

Reply:
@mrshenli 
My mistake must have had a misunderstanding when looking at this issue https://github.com/pytorch/pytorch/issues/46321. I will add a distributed example for the CUDA RPC. To clarify that is the reserved task in the targeted examples?

Reply:
no worries. Yep, it's the reserved task, which links to this page: https://pytorch.org/tutorials/recipes/cuda_rpc.html

Let me add your name to it.